cmd/estat.py

```diff
@@ -91,11 +87,17 @@ def die(*args, **kwargs):
       -d LEVEL  set BCC debug level
       -e        emit the resulting eBPF script without executing it
 
+  estat txg     Collect spa_sync statistics for each txg.
+      -h        show txg help message and exit
+      -c        set the collection interval in seconds
+      -p        set the pool to monitor (default: domain0)
```

Comment:
nit: should include the required argument to these options in the syntax (e.g. akin to `-d LEVEL` above).

Reply:
I added those. 
I also added a header giving an explanation of the standalone tracers.